src/styles/variables.scss

```diff
@@ -6,4 +6,5 @@
 // Breakpoints
 $gt-wide: "only screen and (min-width: 1800px)";
 $lt-tablet: "only screen and (max-width: 1024px)";
+$lt-phone: "only screen and (max-width: 414px)";
```

Comment:
This is not needed for this component, but also below 1024 you can use all the same styles on this site. Meaning, we go to phone view below 1024.

Also, pro tip, an iPhone is actually 750px wide (it displays like it's 375px, but double density makes CSS measure it at 750px). No device in 2020 would ever be 414px wide.